ebean-ddl-generator/src/test/resources/dbmigration/migrationtest/db2/1.1.sql

```diff
@@ -36,16 +36,16 @@ alter table migtest_e_basic alter column status set default 'A';
 alter table migtest_e_basic alter column status set not null;
 alter table migtest_e_basic add constraint ck_mgtst__bsc_stts check ( status in ('N','A','I','?'));
 alter table migtest_e_basic drop constraint ck_mgtst__b_z543fg;
-alter table migtest_e_basic alter column status2 varchar(127);
+alter table migtest_e_basic alter column status2 set data type varchar(127);
 alter table migtest_e_basic alter column status2 drop default;
-alter table migtest_e_basic alter column status2 set null;
+alter table migtest_e_basic alter column status2 drop not null;
```

Comment:
The old statements were wrong. I do not know if they have ever worked.
An `alter table migtest_e_basic alter column status2 set null;` will not work on DB2 11.5 LUW.
@bigjimlefou maybe you can test that on DB2ForI